UI/ChatUI/ChatUI/ChatUI/MainPage.xaml

```diff
@@ -0,0 +1,433 @@
+﻿<Page
+	x:Class="ChatUI.MainPage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:android="http://uno.ui/android"
+	xmlns:behaviors="using:ChatUI.Behaviors"
+	xmlns:converters="using:ChatUI.Converters"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:ios="http://uno.ui/ios"
+	xmlns:local="using:ChatUI"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:xamarin="http://uno.ui/xamarin"
+	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+	mc:Ignorable="d ios android xamarin">
+
+	<Page.Resources>
+		<ResourceDictionary>
+			<converters:FromNullableBoolToVisibilityConverter x:Key="BooleanToVisibility" VisibilityIfTrue="Visible" />
+
+			<Style TargetType="ListView">
+				<Setter Property="RenderTransformOrigin" Value=".5,.5" />
+				<Setter Property="RenderTransform">
+					<Setter.Value>
+						<ScaleTransform ScaleY="-1" />
+					</Setter.Value>
+				</Setter>
+			</Style>
+			<Style TargetType="ListViewItem">
+				<Setter Property="Template">
+					<Setter.Value>
+						<ControlTemplate TargetType="ListViewItem">
+							<Grid
+								x:Name="ContentBorder"
+								Background="{TemplateBinding Background}"
+								BorderBrush="{TemplateBinding BorderBrush}"
+								BorderThickness="{TemplateBinding BorderThickness}"
+								CornerRadius="{TemplateBinding CornerRadius}"
+								RenderTransformOrigin=".5,.5">
+								<Grid.RenderTransform>
+									<!--  Setting the RenderTransform on the ListViewItem itself is not supported by Uno, we need to set it on the first child instead  -->
+									<ScaleTransform ScaleY="-1" />
+								</Grid.RenderTransform>
+								<Rectangle
```

Comment:
Just curious, is there a reason why we're using a rectangle for the background rather just putting the background on the grid below?

Reply:
This was brought over from Steve's POC, I removed it and it doesn't seem to make any difference in the UI.